lib/tokenize.h

```diff
@@ -36,7 +36,7 @@ class TimerResults;
 class Token;
 class TemplateSimplifier;
 class ErrorLogger;
-struct Directive; // IWYU pragma: keep
+struct Directive;
```

Comment:
I removed the pragma since it just partially suppresses the issue. I filed https://github.com/include-what-you-use/include-what-you-use/issues/1541 upstream about the false positive.